### PR TITLE
Add support for configurable products on PPC

### PR DESCRIPTION
**Pull request by mccottry**
# Description
Add suport for custom options on PPC

Fixes: https://app.asana.com/0/941895179897714/1161793187073239

#changelog Add suport for custom options on PPC

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Test

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php [hunk 1]
```diff
@@ -190,10 +190,34 @@ public function isSupportedProductType()
...
+    /**
+     * Get maximum available quantity of product
+     * It is used to show a warning if user tries to buy more then product quantity
+     * @return int -1 Ignore product quantity levels
+     *              0 if product is out of stock
+     *              positive value otherwise
+     *
+     * @deprecated
+     */
+    public function getProductMaxQty()
```
Comment (adentong) [inline app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php]: Why was this function added back? I don't think it's used anywhere.

**Comment by adentong** [comment]
Kinda weird required reviewers weren't added automatically

**Comment by mccottry** [comment]
> Kinda weird required reviewers weren't added automatically

yes. I don't know how that didn't happen.

File: app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php (hunk 1 above)
Comment (mccottry) [reply app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php]: The result of merging with conflicts.  It's a miss on my part.  I'm removing it.

Comment (adentong) [reply app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php]: There were actually no unit tests for this before.

Comment (mccottry) [reply app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php]: ok. there was a bad merge on my end after trying to resolve conflicts.  The method has been removed.

**Comment by johnmccombs1** [comment]
LGTM

**Pull request merged**